com.unity.ml-agents.extensions/Tests/Editor/Sensors/PoseExtractorTests.cs

```diff
@@ -6,18 +6,22 @@ namespace Unity.MLAgents.Extensions.Tests.Sensors
 {
     public class PoseExtractorTests
     {
-        class UselessPoseExtractor : PoseExtractor
+
+        class BasicPoseExtractor : PoseExtractor
         {
             protected internal override Pose GetPoseAt(int index)
             {
                 return Pose.identity;
             }
 
-            protected internal override Vector3 GetLinearVelocityAt(int index)
+            protected  internal override Vector3 GetLinearVelocityAt(int index)
```

Comment:
```suggestion
            protected internal override Vector3 GetLinearVelocityAt(int index)
```

Reply:
I'll do a pass with the code formatter in another PR - it had made some whitespace changes in the actuator code too but I reverted them.